runtime/lua/vim/_editor.lua

```diff
@@ -31,6 +31,7 @@ for k, v in pairs({
   loader = true,
   func = true,
   F = true,
+  img = true,
```

Comment:
`vim.ui.img` seems like the right place for this (else why do we have `vim.ui`? :)

Reply:
@justinmk sure, happy to put it there. Would you like me to refactor this PR to have it placed under the namespace?

Reply:
yes